test/ia-menu-slider.test.js

```diff
@@ -138,4 +138,81 @@ describe('<ia-menu-slider>', () => {
 
     expect(el.selectedMenu).to.not.equal(menus[2].id);
   });
+
+  describe('Header section', async () => {
+    let extraButtonClicked = false;
+    const extraActionClickHandler = () => {
+      extraButtonClicked = true;
+    };
+    const menu = {
+      icon: html`<ia-icon icon="audio"></ia-icon>`,
+      title: 'Audio Menu',
+      menuDetails: '(100,000,000 tracks)',
+      actionButton: html`
+        <button
+          style='color: white;'
+          @click=${extraActionClickHandler}
+        >Foo
+        </button>
```

Comment:
SUGGESTION/QUESTION: If you're going to make sure to put `Foo` on the line with the `>` to make sure it has no whitespace before `Foo`, then perhaps you want to put `</button>` directly after `Foo` so there's also no whitespace after `Foo`...?

Reply:
thanks, moved it up.